sdk-config.json

```diff
@@ -2,7 +2,7 @@
   "comment_sdk_config": "This is the configuration file for the Angular-SDK",
   "authConfig": {
     "authConfig_comment": "Optional Full paths to Infinity server OAuth endpoints (Only needed if using custom OAuth service)",
-    "authConfig_comment2": "Optional attributes are: authorize, token, revoke, authService, silentTimeout, iframeLoginUI",
+    "authConfig_comment2": "Optional attributes are: authorize, token, revoke, redirectUri, authService, silentTimeout, iframeLoginUI",
```

Comment:
@syedi2 - Not sure if you know about this additional (optional) param that needs to be documented. I think you saw this for SDK-R but now being added to SDK-A, too.